checker/bin/wpi.sh

```diff
@@ -291,22 +293,26 @@ elif [ "${has_java16}" = "yes" ]; then
 fi
 configure_and_exec_dljc "$@"
 
+# If running under Java 8 failed and Java 11 is also available, then try running again under Java 11.
```

Comment:
The logic is right, but I had to read it several times to convince myself of that.
This sentence here lays out the logic, but the `if` statements take a different approach than explained in the sentence.
How about making the `if` statements closer to the English, such as grouping
`"${has_java8}" = "yes" && "${WPI_RESULTS_AVAILABLE}" != "yes"`
together since it expresses "running under Java 8 failed"?

Reply:
I both re-wrote the comments and re-ordered the tests, so that the two match up exactly. Can you take another look?